GSoC-interest.md

```diff
@@ -17,7 +17,8 @@ GSoC Mentors
 
 
 | Name | Email | Idea | Repo | Project Proposal |
-| --- | --- | --- | --- | --- |
```

Comment:
Please, don´t remove this line, and add your entry after the one for Georg, which is an example. Preferably, add it the latest one. Thanks!

Reply:
Sorry about that @jgbarah